palantirnet/the-vagrant
# Option to install the drupal-vm

Notes so far...

1. setup time is > 10 minutes (13 mins for me this time, but it will depend on your connection)... but the first time it's more like 35 minutes because you have to download the base box
2. I think this installs and configures solr, but we might also have to start the solr service? not sure, but I can't access solr at http://projectname.drupalvm:8983/solr

To test:

* On a project, require this branch of `the-vagrant` (please don't do this permanently, as this branch will be deleted if the PR is merged) -- `composer require --dev palantirnet/the-vagrant:dev-drupal-vm`
* Destroy your existing box: `vagrant destroy`
* Install the drupal-vm: `vendor/bin/the-vagrant-installer-drupalvm`
  * This will prompt you for a project name, which defaults to the name of the project directory
* Observe two new/updated files: `Vagrantfile` and `conf/drupalvm/config.yml`
* Bring up your vagrant box: `vagrant up` (this will take 10-45 minutes)
* Log in to your vagrant: `vagrant ssh`
* Find your project root: `cd /var/www/drupalvm` and observe your project stuff there
* You will need to update your mysql db, username, and password in your `conf/build.default.properties` file to use `drupal`/`drupal`/`drupal`
* Try running drush and phing and stuff
* Visit your site in a browser: http://projectname.drupalvm
